lib/tagging_migrator.rb

```diff
@@ -0,0 +1,58 @@
+# TaggingMigrator
+#
+# We are migrating the taggings from panopticon/content-api to publishing-api.
+# This class takes the taggings for a certain app and sends it to the
+# publishing-api so that it contains the current taggings. This functionality
+# will be removed after the migration.
+
+require 'gds_api/publishing_api_v2'
+
+class TaggingMigrator
+  def initialize(app_name)
+    @app_name = app_name
+  end
+
+  def migrate!
+    artefacts_owned_by_app.each do |artefact|
+      next unless artefact.content_id
```

Comment:
Happy to drop all these - they're not affecting the live site, just old left-behind data.